bin/init.bash

```diff
@@ -86,15 +86,20 @@ generate_sops_config() {
     sops_config_write_fingerprints "${fingerprint}"
 }
 
-# Only writes value if it is set to "set-me*"
+# Only writes value if it is set to "set-me", or a conditional "set-me" and the condition is true
 # Usage: replace_set_me <file> <field> <value>
 replace_set_me(){
     if [[ $# -ne 3 ]]; then
         log_error "ERROR: number of args in replace_set_me must be 3. #=[$#]"
         exit 1
     fi
-    if [[ $(yq4 "${2}" "${1}") =~ ^set-me.* ]]; then
+    if [[ $(yq4 "${2}" "${1}") == "set-me" ]]; then
         yq4 --inplace "${2} = ${3}" "${1}"
+    elif [[ $(yq4 "${2}" "${1}") =~ ^set\-me\-if\-.*$ ]]; then
```

Comment:
Do the dashes need to be escaped?

Reply:
Maybe not. In any case, this function is not used anymore after the config template rework, so I'll just remove it.